src/commands/add-formatting.js

```diff
@@ -0,0 +1,62 @@
+const hljs = require('highlight.js');
+const confusedMessageGenerator = require('../utilities/confused-message-generator');
+const processReply = require('../utilities/process-reply');
+
+module.exports = {
+  prefix: '',
```

Comment:
This command is triggered by the any user's `bot`  emoji reaction to any message.

Reply:
The prefix doesn't really apply to this command.... infact, I don't know if we can call it a command at all since it listens for reaction rather than messages. Maybe move it to a separate directory all together?

Reply:
Yea I'd move this file somewhere else, since its not a command.